docs/control-structures.rst

```diff
@@ -2,13 +2,14 @@
 Expressions and Control Structures
 ##################################
 
-.. index:: ! parameter, parameter;input, parameter;output, function parameter, parameter;function, return variable, variable;return, return
+.. index:: ! parameter, parameter;input, parameter;output, function parameter, parameter;function, return variable, variable;return, return value, value;return, return
 
 Function Parameters and Return Variables
 ========================================
 
 As in JavaScript, functions may take parameters as input. Unlike in JavaScript
-and C, functions may also return an arbitrary number of values as output.
+and C, functions may also return an arbitrary number of variables that each
```

Comment:
I would argue that what the functions return is actually a value - the variable does not exist after the function has returned. If the function returns a variable, it might sound like the variable exists after the function returned.

Reply:
I.e. I would propose to keep this unchanged or just remove the "as output".